.github/workflows/release.yaml

```
@@ -54,10 +56,26 @@ jobs:
           mv paranoia-linux-arm64/paranoia bin/paranoia-linux-arm64
           mkdir man
           mv man-pages/* man/
+      - name: Generate hashes
+        shell: bash
+        id: hash
+        run: |
+          echo "hashes=$(sha256sum bin/paranoia-darwin-amd64 bin/paranoia-darwin-arm64 bin/paranoia-linux-amd64 bin/paranoia-linux-arm64 | base64 -w0)" >> "$GITHUB_OUTPUT"
       - name: Release
         uses: softprops/action-gh-release@v1
         with:
           files: |
             LICENSE.txt
             bin/paranoia-*
             man/*
+  provenance-for-binaries:
```

nit: add a blank line between jobs
```suggestion

  provenance-for-binaries:
```